app/assets/stylesheets/queries.css.sass

```diff
@@ -111,3 +111,6 @@ tags-input .tags .tag-item
 
 .result-panel
   margin-bottom: 0px
+
+.group-query-items
```

Comment:
why did you call it this? seems more like `column-spacer` or something right?

Reply:
This class name should be an indication that it will keep author, created_at, updated_at items together in col-md-4 div in responsive resolutions (avoiding #54)
Obviously not a sufficiently intuitive, I'll try to find some better name